execution_workflows/QAPA/more_samplesheets/samplesheet_paired.csv

```diff
@@ -0,0 +1,2 @@
+sample,fastq1,fastq2,bam,bai,gff,fasta,bed,mart_export
+test,/home/wanyk/Downloads/apa/fastq/SRR3067957_1.fastq,/home/wanyk/Downloads/apa/fastq/SRR3067957_2.fastq,/home/wanyk/Downloads/apa/bam/Aligned.out.sorted.bam,/home/wanyk/Downloads/apa/bam/Aligned.out.sorted.bam.bai,/home/wanyk/Downloads/apa/gencode.vM26.primary_assembly.annotation.gff3,/home/wanyk/Downloads/apa/mm39.fa,/home/wanyk/Downloads/apa/qapa_3utrs.gencode_VM22.mm10.bed,/home/wanyk/Downloads/apa/mart_export.txt
```

Comment:
Best to update this with s3 links once we have them. Also, it's not considered best security practices to have absolute paths to files in a repo (which anyway wouldn't be portable). Goes for file paths in other files, too

Reply:
I think I have seen `nf-core/viralrecon` staging files from s3 from a `[name].config` file (https://github.com/nf-core/viralrecon/blob/master/conf/test_full_nanopore.config). I can look into how to do that.